.github/workflows/python-code-quality.yml

```diff
@@ -146,3 +150,11 @@ jobs:
           name: sphinx-grass
           path: sphinx-grass
           retention-days: 3
+  python-success:
+    name: Python Code Quality Result
```

Comment:
What do you think about directly skipping to using something more generic like "Linting Result", so we could merge other code checking jobs in this without having to go back for a round of config changes, only a PR to approve?

Yesterday, I was imagining that it would be in this file that I would wire up Ruff, and I'm thinking about Clang Format too since it is kind of related.

Reply:
I guess it makes sense. Perhaps the diagram will be put to some good use while the top level will stay clean. I did a lot of the check-per-tool workflows where both the CI and the use of strict CI were new in the project. Now, everyone is used to CI complaining in all projects and pre-commit moves the checking back, closer to the author, so I expect that contributors can just investigate if needed.

Reply:
Let's leave this for another PR.